projects/fsastorefrontlib/src/cms-components/assets-table/assets-table.component.ts

```diff
@@ -14,8 +15,31 @@ import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class AssetsTableComponent {
-  constructor() {}
+  constructor(protected routingService: RoutingService) {}
 
   @Input() headings: string[];
   @Input() assets: { [key: string]: any }[];
+  @Input() isSeller: boolean;
+
+  resolveAssetUrl(asset: { [key: string]: any }) {
```

Comment:
If user is seller, redirection is disabled, otherwise, user is redirected to requested asset